gcm/connector.go

```diff
@@ -0,0 +1,233 @@
+package gcm
+
+import (
+	log "github.com/Sirupsen/logrus"
+	"github.com/alexjlockwood/gcm"
+	"github.com/smancke/guble/protocol"
+	"github.com/smancke/guble/server"
+	"github.com/smancke/guble/store"
+	"strconv"
+
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+	"sync"
+)
+
+const (
+	// registrationsSchema is the default sqlite schema for GCM
+	schema = "gcm_registration"
+
+	// sendRetries is the number of retries when sending a message
+	sendRetries = 5
+
+	subscribePrefixPath = "subscribe"
+
+	// default channel buffer size
+	bufferSize = 1000
+)
+
+var logger = log.WithField("module", "gcm")
```

Comment:
var logger = log.WithFields(log.Fields{
	"app":    "guble",
	"module": "gcm",
	"env":    "TBD",
}) 

this is the common usage